Extension/src/LanguageServer/client.ts

```diff
@@ -2724,15 +2728,22 @@ export class DefaultClient implements Client {
 
         showConfigStatus = showConfigStatus || (configurationNotSet &&
             !!compilerDefaults && !compilerDefaults.trustedCompilerFound && trustedCompilerPaths && (trustedCompilerPaths.length !== 1 || trustedCompilerPaths[0] !== ""));
+        if (showConfigStatus) {
+            this.showConfigureIntelliSenseButton = true;
```

Comment:
This may cause bugs in which the configuration status will appear even if the statusBarIndicatorEnabled is false. You should only set this if statusBarIndicatorEnabled is true, because other code that handles the activeDocumentChanged will check this variable. You can use the original check and just pass in showConfigStatus as the 1st param.

Reply:
Actually, I don't think this causes a bug, but it'd still be safer to not change the this.showConfigureIntelliSenseButton state when not in a mode where that is used.

Reply:
reverted this